### DISCUSSION TITLE
google AdSense for website hosting on github pages for free

### DISCUSSION THREAD

**Discussion by samstack-sujaldey** [community/community#102988] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

can i apply for google adsense while hosting website on github

**Comment by DaviZCodes** [comment, marked as answer]
Yes, you definitely can as long as you follow Google Ad Sense Rules and GitHub rules for ads. 

Look at this example: https://pscustomobject.github.io

**Reply by onacit** [reply]
Am I supposed to see ads on the site? I don't think I'm seeing any...

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @sujaldey004! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!

**Comment by Ismael9-web** [comment]
really?
